code/datums/client_image_group.dm

```diff
@@ -0,0 +1,90 @@
+var/global/list/datum/client_image_group/client_image_groups
+
+/datum/client_image_group
+	var/list/image/images
+	var/list/loc_to_image_lookup
+	var/list/subscribed_mobs
+
+	New()
+		. = ..()
+		images = list()
+		loc_to_image_lookup = list()
+		subscribed_mobs = list()
+
+	proc/add_image(image/img) // adds to the list and adds to all mobs clients directly
```

Comment:
why didn't you just use doc comments here

Reply:
Refactored to use doc comments and improved their contents.